src/main/java/seedu/revised/exception/flashcard/NoFlashcardException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.revised.exception.flashcard;
+
+public class NoFlashcardException extends Exception {
```

Comment:
Inside the constructor of the classes that extends Exception, it should call super(message). In this case, it should call super(printNoFlashcardsError).